mifosng-android/src/main/AndroidManifest.xml

```diff
@@ -26,18 +26,16 @@
         <meta-data
             android:name="com.google.android.gms.version"
             android:value="@integer/google_play_services_version" />
-<!--        <meta-data-->
-<!--            android:name="io.fabric.ApiKey"-->
-<!--            android:value="facae47316396ae7954149cc408e25b7a37ab7ea" />-->
```

Comment:
Open an issue to implement the firebase inplace of fabric, upgrade fabric to firebase